voxblox/CMakeLists.txt

```diff
@@ -165,6 +166,11 @@ catkin_add_gtest(test_layer
 )
 target_link_libraries(test_layer ${PROJECT_NAME} ${catkin_LIBRARIES})
 
+catkin_add_gtest(test_layer_utils
+  test/test_layer_utils.cc
+)
+target_link_libraries(test_layer_utils ${PROJECT_NAME} ${catkin_LIBRARIES})
```

Comment:
I don't think you actually need ${catkin_LIBRARIES} here, should be done by catkin simple.

Reply:
Hmm but add gtest isn't a cs command? Not sure on this...

Reply:
I guess this is a more general discussion, not limited to this PR :)
https://github.com/ethz-asl/voxblox/issues/134